keylime/elchecking/tests.py

```diff
@@ -10,11 +10,13 @@
 # A Test can be used multiple times, even concurrently.
 
 # Data is the type of Python data that corresponds to JSON values.
-Data = typing.Union[int, float, str, bool, typing.Tuple["Data", ...], typing.Mapping[str, "Data"], None]
+Data = typing.Union[
+    int, float, str, bool, typing.Tuple["Data", ...], typing.Mapping[str, "Data"], typing.List["Data"], None
```

Comment:
Here and a bit later in line 19 we refer again the ABC instead the type. Is there a real advantage?

Reply:
I don't know. What would be your suggestion?